index.js

```diff
@@ -184,7 +185,7 @@ HTTP_RGB.prototype = {
                     .on('set', this.setLockTarget.bind(this));
 
                 return [lockService];
-
+                
```

Comment:
Extra Spaces.

Reply:
Done, but I'm not totally sure if I got it. 
Please check if the changes are like you requested.